README.md

```diff
@@ -92,7 +92,7 @@ For example the cards layout appears by using the `<Callout>` component that can
 You can use this component within your content as follows:
 
 ```js
-<Callout type="note">Cody is currently in Beta stage for all users.</Callout>
+<Callout type="note">Cody is currently in Beta for all users.</Callout>
```

Comment:
```suggestion

```
We should remove this as Cody is not in Beta anymore.

Reply:
Ah true, but I think this is just an example in the README for how to format `callout` components. This change is just to make the javascript example match the image above

Reply:
Oh, my bad, I just missed this context. 🙈